gp-includes/template.php

```diff
@@ -105,7 +105,10 @@ function gp_nav_menu_items( $location = 'main' ) {
 	}
 	elseif ( 'side' === $location ) {
 		if ( is_user_logged_in() ) {
-			$items[ gp_url( '/profile' ) ] = __( 'Profile', 'glotpress' );
+			$user = wp_get_current_user();
+			$login = esc_html( $user->user_login );
+			$items[ gp_url( '/profile/' . $login ) ] = __( 'Profile', 'glotpress' );
```

Comment:
This should use `gp_url_profile()`. Note that the slug of a user is `user_nicename`, not `user_login`.